packages/edit-site/src/components/dataviews/add-filter.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	privateApis as componentsPrivateApis,
+	Button,
+	BaseControl,
+	Icon,
+} from '@wordpress/components';
+import { chevronRightSmall } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const {
+	DropdownMenuV2,
+	DropdownSubMenuV2,
+	DropdownSubMenuTriggerV2,
+	DropdownMenuItemV2,
+} = unlock( componentsPrivateApis );
+
+export default function AddFilter( { filters, onChangeView } ) {
+	return (
+		<BaseControl>
+			<DropdownMenuV2
+				label={ __( 'Add filter' ) }
+				trigger={
+					<Button disabled={ ! filters?.length } variant="tertiary">
+						{ __( '+ Add filter' ) }
```

Comment:
Should the `+` be an icon here?

Reply:
What'd be the advantage of using an icon? It seems that using it as a string would allow different languages to position it differently (should it be necessary)?

Reply:
Used at icon at https://github.com/WordPress/gutenberg/pull/55992/commits/9c99e4f3c97cc01892036b0574fe489137646788 I figured UI stability is preferred for the `+` across languages.